src/core/qgsvectorlayerfeatureiterator.h

```diff
@@ -286,7 +286,7 @@ class CORE_EXPORT QgsVectorLayerFeatureIterator : public QgsAbstractFeatureItera
     /**
      * Checks a feature's geometry for validity, if requested in feature request.
      */
-    bool checkGeometryValidity( const QgsFeature &feature );
+    bool checkGeometryValidity( QgsFeature &feature );
```

Comment:
Can we rename thais to `handle` and adjust docstrings

Reply:
you mean to renamed it to 'handleGeometryValidity' ? will this cause any API breaks?

Reply:
Isn't it private?

Reply:
Ah right its internal only. I'll change it right away then!